### PR TITLE
Adding client tests for fetching tokens

### PR DESCRIPTION
**Pull request by leikoilja**
Haven't really had time to finish it up with `homegraph` and `get_devices`, but added some client tests to make sure we fetch tokens right.

### CODE REVIEW

File: glocaltokens/client.py [hunk 1]
```diff
@@ -145,9 +145,9 @@ def get_android_id(self) -> Optional[str]:
         return self.android_id
 
     @staticmethod
-    def _token_has_expired(token_date, token_duration):
-        """Checks if an specified token has expired"""
-        return datetime.now().timestamp() - token_date.timestamp() > token_duration
+    def _has_expired(creation_dt, duration):
```
Comment (ArnyminerZ) [inline glocaltokens/client.py:148]: Makes sense. Please, use typing. Maybe `_has_expired(creation_dt: datetime, duration: int) -> bool`? Not sure about that

**Review by ArnyminerZ — CHANGES_REQUESTED** [review]
Awesome work! Great naming changes, and incredible testing 😮

File: glocaltokens/client.py (hunk 1 above)
Comment (leikoilja) [reply glocaltokens/client.py:148]: Oh right, sorry, completely forgot about typing. Pretty new concept for my in python, so thanks for reminding :)

**Comment by leikoilja** [comment]
Thanks, @ArnyminerZ! 💥  I have just added homegraph test too, which is pretty simple i guess.
Do you want to try taking the `get_devices` and `get_devices_json` methods to test? It could be a fun practise, if you are up for a challenge 💯 No worries if you are busy with something, we have a ticket for it (https://github.com/leikoilja/glocaltokens/issues/23), so I ll just pick it up a bit later

**Comment by ArnyminerZ** [comment]
Ok, I can try developing them by seeing your work. Will take a try 😉

**Review by ArnyminerZ — APPROVED** [review]

**Comment by leikoilja** [comment]
@ArnyminerZ sounds amazing! Feel free to make another branch and reference #30 issue there. You can also always push your work into a branch, open a PR and prefix it's title with [WIP] (standing for "work in progress"), then you can ping me or anyone else in the comments in that PR and ask help if needed 💥

**Pull request merged**